apply.php

```diff
@@ -13,61 +13,57 @@
 </head>
 
 <body>
-	<?php include 'navbar.php'; ?>
-	<div id="myCarousel" class="carousel slide" data-ride="carousel">
-		<div class="carousel-inner">
-			<img src="imgs/coffee12.png" width="100%">
-			<div class="item active">
-				<div class="carousel-caption d-none d-md-block">
-					<div class="hero-text">
-						<h2>Become a Part of the StarBulls</h2>
-						<p>Apply Today!</p>
+	<div class="container-fluid p-0">
+		<?php include 'navbar.php'; ?>
+		<div id="myCarousel" class="carousel slide" data-ride="carousel">
+			<div class="carousel-inner">
+				<img src="imgs/coffee12.png" width="100%">
+				<div class="item active">
+					<div class="carousel-caption d-none d-md-block">
+						<div class="hero-text">
+							<h2>Become a Part of the StarBulls</h2>
+							<p>Apply Today!</p>
+						</div>
 					</div>
 				</div>
 			</div>
 		</div>
-	</div>
-	<div id="columns">
-		<div class="column-wrap">
-			<div class="column-left">
-				<!--left content-->
-				<h2>Application</h2>
-				<h3>*required fields</h3>
-				<div class="form-container">
-					<?php include 'forms/apply.form.php'; ?>
+		<div id="applyContent">
+			<div class="column-wrap">
+				<div class="apply-column-left">
+					<!--left content-->
+					<h2>Application</h2>
+					<h3>*required fields</h3>
+					<div class="form-container">
+						<?php include 'forms/apply.form.php'; ?>
+					</div>
+				</div>
+			</div>
```

Comment:
Headline for Reviews is appearing at the foot of the left column. I think it's an extra closing div on line 41.
![image](https://user-images.githubusercontent.com/78501210/112761863-38dbe400-8fcb-11eb-9644-dc6e2344d08f.png)

Reply:
So odd, I deleted the extra div but it still didn't move over, went ahead and just reclassified the divs with bootstrap.